game.py

```diff
@@ -0,0 +1,98 @@
+# Here is the description of the game simulation.
+
+# this is a two player card game
+# the game starts with a deck of cards
+# the cards are dealt out to both players
+# on each turn:
+# both players turn over their top-most card
+# the player with the higher valued card takes the cards and puts them in their scoring pile (scoring 1 point per card)
+# this continues until the players have no cards left
+# the player with the highest score wins
+# It’s considered a simulation because the players don’t have any choice, dont worry about input
+import random
+
+class CardDeck():
+    suits = ['spade', 'heart', 'diamond', 'club']
+    vals = [i for i in range(1,14)]
```

Comment:
These will be class variable shared between all objects.
If you have multiple objects and then accidentally modify this once, then every object will have the modified one. This would be difficult to debug.